R/fit_process.R

```diff
@@ -312,78 +303,6 @@ calculate_lancelot_ifr_t <- function(samples, multistrain) {
 
 ## All the functions below here have awful names, but none are
 ## exported so we can tidy this up later.
-extract_outputs_by_age <- function(sample, what) {
```

Comment:
Always nice to see old code get deleted :)

Reply:
Absolutely liberating!